llvm/include/llvm/Analysis/IR2Vec.h

```diff
@@ -24,6 +24,12 @@
 ///   Code Optimization (TACO), 2020. https://doi.org/10.1145/3418463.
 ///   https://arxiv.org/abs/1909.06228
 ///
+/// To obtain embeddings:
+/// First run the IR2Vec vocabulary analysis pass to populate the vocabulary.
```

Comment:
`s/the IR2Vec vocabulary analysis pass/<the actual name of the class>`

... so that one can find it fast in code